### PR TITLE
V3.0.0

### PR DESCRIPTION
**Pull request by jonhealy1**

### CODE REVIEW

**Comment by jonhealy1** [comment]
@gadomski @jbants There is this test ```Execute tests (3.6) Expected — Waiting for status to be reported``` that never seems to finish and I don't know how to remove it - it's really annoying. Is it some sort of an artifact?

**Review by jbants — APPROVED** [review]
Looks good. 
Thanks Jon.

**Comment by jbants** [comment]
> @gadomski @jbants There is this test below `Execute tests (3.6) Expected — Waiting for status to be reported` that never seems to finish and I don't know how to remove it - it's really annoying. Is it some sort of an artifact?

I think this is because it still references the .tox file in `master`. Merging this should fix the test. 
This Pr should close #189

**Comment by gadomski** [comment]
> @gadomski @jbants There is this test below Execute tests (3.6) Expected — Waiting for status to be reported that never seems to finish and I don't know how to remove it - it's really annoying. Is it some sort of an artifact?

It was a leftover check for a python 3.6 CI run that got removed. I removed the check from the branch protection and things look good now.

**Comment by jonhealy1** [comment]
> > @gadomski @jbants There is this test below Execute tests (3.6) Expected — Waiting for status to be reported that never seems to finish and I don't know how to remove it - it's really annoying. Is it some sort of an artifact?
> 
> It was a leftover check for a python 3.6 CI run that got removed. I removed the check from the branch protection and things look good now.

Nice thank you!!!

**Comment by jonhealy1** [comment]
https://github.com/stac-utils/stac-validator/issues/196 I don't know if this works until this pr is merged ...

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 
 The format is (loosely) based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [3.0.0] - 2022-03-11
```
Comment (gadomski) [inline CHANGELOG.md:7]: Link needs to be added to bottom of CHANGELOG.

**Review by gadomski — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jonhealy1) [reply CHANGELOG.md:7]: I think I fixed this and did this right but I am not totally sure ...

**Pull request merged**